src/main/java/com/github/wovnio/wovnjava/HtmlChecker.java

```diff
@@ -0,0 +1,59 @@
+package com.github.wovnio.wovnjava;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+class HtmlChecker {
+    public boolean canTranslateContentType(String type) {
+        return type == null || type.toLowerCase().contains("html");
+    }
+
+    public boolean canTranslatePath(String path) {
+        if (path == null) {
+            return true;
+        }
+
+        // Reduce strings for performance and keep a simple case
+        path = path.replaceFirst("^.*/", "/"); // strip directries
+        path = path.replaceFirst("[?#].*$", ""); // strip query or/and hash
+        path = path.toLowerCase();
+
+        return !IMAGE_FILE_PATTERN.matcher(path).find()
+            && !AUDIO_FILE_PATTERN.matcher(path).find()
+            && !VIDEO_FILE_PATTERN.matcher(path).find()
+            && !DOC_FILE_PATTERN.matcher(path).find();
+    }
+
+    public boolean canTranslateContent(String html) {
+        if (html == null) {
+            return false;
+        }
+
+        String head = getHead(html).toLowerCase();
+        return isHtml(head) && !isAmp(head);
+    }
+
+    private boolean isAmp(String head) {
+        return head.contains("<html ⚡")
```

Comment:
If there is other attributes in HTML tag, then this check could fail.
Example:
```
<html lang="en" amp>
```

Reply:
That's right.
I fixed it.